version.py

```diff
@@ -34,7 +34,7 @@
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
 
-API_FEATURE_LEVEL = 343  # Last bumped for `stream_ids` field in stream deletion events.
+API_FEATURE_LEVEL = 344  # Last bumped for allowing channel admins to unsubscribe users.
```

Comment:
Not too sure about this bump, I've always done it in the past when there was a change to a field in the API, but I think we still might need to do since the API behaviour is changing significantly after this PR?

Reply:
I think it's never a bad idea to bump for a significant behavior change that might impact how clients are implemented.